src/cache/colony/index.ts

```diff
@@ -0,0 +1,13 @@
+import tokensField from './tokens';
+import balancesField from './balances';
+
+const colony = {
+  Colony: {
+    fields: {
```

Comment:
You mentioned apollo only permits 3 local fields. What's the third one stopping us from doing the claims logic here too? (re your comment about the claims logic (`@NOTE We have to do some very heavy lifting...`)

Reply:
That's exactly the problem, any other field I add to that object, it crashes everything.